src/features/mods/ModLibraryPage.css

```
@@ -498,13 +498,43 @@
+  font-weight: 700;
+  line-height: 1.35;
+}
+
+.compact-panel__selection-announcement {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)